src/components/Tag/Tabs/VulnerabilitiesDetails.jsx

```diff
@@ -172,6 +241,12 @@ function VulnerabilitiesDetails(props) {
     };
   }, []);
 
+  useEffect(() => {
+    if (openExport) {
```

Comment:
maybe also add a secondary condition to check if allCveData is empty here, so we avoid repeated calls to the backend if we already fetched the data previously.